src/components/About/about.css

```diff
@@ -0,0 +1,106 @@
+.about-wrapper {
+  width: 340px;
+  margin: 0 auto;
+  border-radius: 20px;
+  background-color: var(--primary-element-color);
+}
+
+.about-title {
+  width: 300px;
+  display: block;
+  margin: 0 auto;
+  padding-top: 20px;
+  padding-bottom: 20px;
+  color: var(--primary-text-color);
+}
+
+.about-text {
+  width: 300px;
+  height: auto;
+  margin: 0 auto;
+  color: var(--secondary-text-color);
+}
+
+.about-text a {
+  word-wrap: break-word;
+  transition: none;
```

Comment:
this 2 rules say us: we have problem in another place. Links by default don't have problems with lines. Transition... why do we have it here?

Reply:
done